victorialogs: marked fluentd support in roadmap, added syslog example (#7098)
VictoriaMetrics/VictoriaMetrics: deployment/docker/victorialogs/fluentbit/README.md

The docker compose file contains the following components:

> * fluentbit - FluentBit logs collection agent, which is configured to collect and write data to `victorialogs`

Reviewer: ```suggestion
* fluentbit - logs collection agent configured to collect and write data to `victorialogs`
```

> * victorialogs - VictoriaLogs log database, which accepts the data from `fluentbit`

Reviewer: ```suggestion
* victorialogs - logs database, receives data from `fluentbit` agent
```

> * victoriametrics - VictoriaMetrics metrics database, which collects metrics from `victorialogs` and `fluentbit`

Reviewer: ```suggestion
* victoriametrics - metrics database, collects metrics from `victorialogs` and `fluentbit` for observability purposes
```
